common/persistence/src/main/java/app/coronawarn/server/common/persistence/service/LocalStatisticsDownloadService.java

```diff
@@ -0,0 +1,72 @@
+package app.coronawarn.server.common.persistence.service;
+
+import static java.time.ZoneOffset.UTC;
+
+import app.coronawarn.server.common.persistence.domain.LocalStatisticsDownload;
+import app.coronawarn.server.common.persistence.repository.LocalStatisticsDownloadRepository;
+import java.time.Instant;
+import java.time.LocalDateTime;
+import java.util.Optional;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.stereotype.Component;
+import org.springframework.transaction.annotation.Transactional;
+
+@Component
+public class LocalStatisticsDownloadService {
+
+  private final LocalStatisticsDownloadRepository repository;
+  private static final Logger logger = LoggerFactory.getLogger(LocalStatisticsDownloadService.class);
+
+  public LocalStatisticsDownloadService(LocalStatisticsDownloadRepository repository) {
+    this.repository = repository;
+  }
+
+  /**
+   * Stores an entry in the Database with {@param timestamp} as seconds and {@param eTag}. The counter is an incremental
+   * value automatically determined by a postgres sequence.
+   *
+   * @param timestamp in seconds.
+   * @param etag      value to be stored.
+   * @return boolean determining if store was successful or not.s
+   */
+  @Transactional
+  public boolean store(long timestamp, String etag) {
+    try {
+      this.repository.insertWithAutoIncrement(timestamp, etag);
```

Comment:
I think also the insert statement of the repository can return an boolean which indicates if the insertion was successful

Reply:
Agree. Refactored it.